File: app/src/main/java/com/martin/preventapp/ui/new_order/recyclerView/CardViewOrder.java
package com.martin.preventapp.ui.new_order.recyclerView;

public class CardViewOrder {

    private String product;
    private String amount;
    private String unit;
    private String positionItem;

    public CardViewOrder(String product2, String amount2, String unit2, String positionItem2) {
        product = product2;
        amount = amount2;
        unit = unit2;
        positionItem = positionItem2;
    }

    public String getProduct() {
        return product;
    }

    public void setTextProduct(String product2) {
        product = product2;
    }

    public String getAmount() {
        return amount;
    }

    public void setTextAmount(String amount2) {
        amount = amount2;
    }

    public String getUnit() {
        return unit;
    }

    public void setUnit(String unit2){
        unit = unit2;
    }

    public String getPositionItem() {
        return positionItem;
    }

    public void setPositionItem(String positionItem) {
        this.positionItem = positionItem;
    }
}

File: app/src/main/java/com/martin/preventapp/View/new_order/recyclerView/CardViewOrder.kt
package com.martin.preventapp.View.new_order.recyclerView

class CardViewOrder(
    var product: String,
    var amount: String,
    var unit: String,
    var positionItem: String
) {
    fun setTextProduct(product2: String) {
        product = product2
    }

    fun setTextAmount(amount2: String) {
        amount = amount2
    }
}